Makefile

```diff
@@ -11,7 +11,11 @@ export GIT_LS_FILES ?= git ls-files --cached --others --exclude-standard
 # See https://docs.python.org/3/using/cmdline.html#envvar-PYTHONUSERBASE
 export PYTHONUSERBASE ?= $(HOME)/.local
 
+ifdef VIRTUAL_ENV
```

Comment:
Can you add a test for this or is that not possible?

Reply:
done!